Simplenote/src/androidTest/java/com/automattic/simplenote/smoke/utils/TestUtils.java

```diff
@@ -0,0 +1,30 @@
+package com.automattic.simplenote.smoke.utils;
+
+import com.automattic.simplenote.smoke.pages.IntroPage;
+import com.automattic.simplenote.smoke.pages.LoginPage;
+import com.automattic.simplenote.smoke.pages.MainPage;
+
+public class TestUtils {
+
+    private static final int GIVE_ME_A_BREAK_TIME = 2000;
+
+    public static void idleFor(int milliseconds) {
+        try {
+            Thread.sleep(milliseconds);
+        } catch (Exception ex) {
+            // do nothing
+        }
+    }
+
+    public static void giveMeABreak() {
```

Comment:
This is an odd method name – could we either use something more standard, or just use `idleFor` directly?

Reply:
I will handle it in a better and regular naming.